tasks/minesweeper/README.md

```diff
@@ -15,25 +15,27 @@ You are to implement a classic game — [Minesweeper](<https://en.wikipedia.org/
 - initially, `body` in the index.html file must be empty (only script tag is allowed), **all necessary elements are generated using JS**
 - the design should be adaptive (or responsive) from _(500px <= width)_. It is acceptable to change the appearance for the mobile version (for example, hide the buttons in the burger menu)
 - the default size of the frame is 10x10 with 10 mines.
+- the game starts after user made first click using left button on mouse
 - the player should be able to click on cells to reveal them. If the cell contains a mine, the game is over. If the cell does not contain a mine, the number of mines in the surrounding cells should be displayed
 - the game should end when the player reveals all cells that do not contain mines (win) or clicks on mine (lose). On a successful game solution, display the message: "Hooray! You found all mines in ## seconds and N moves!" or "Game over. Try again"
 
 `Advanced:`
 
-- mines are placed after the first move, so that user cannot lose the game on the first move
+- mines are placed after the first move, so that user cannot lose the game on the first move. It means that during the first move, the user can either open one cell with a number or empty cells and the nearest cells with numbers if this item in the specifications is implemented.
 - the player should be able to flag cells to indicate that they suspect a mine is present
```

Comment:
Maybe it would be good to clafiry **how** user can place flag (e.g. _"The user can place a flag on a cell by right-clicking on it"_)?

Reply:
fixed